pkg/synthetictests/etcd.go

```diff
@@ -0,0 +1,36 @@
+package synthetictests
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/openshift/origin/pkg/monitor/monitorapi"
+	"github.com/openshift/origin/pkg/test/ginkgo/junitapi"
+)
+
+func testEtcdShouldNotLogSlowFdataSyncs(events monitorapi.Intervals) []*junitapi.JUnitTestCase {
```

Comment:
Problem here, this appears to be quite common, I already see it happening on gcp in the presubmits the first time I had it working... Is the test of any use? Do we allow no more than one? https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/27772/pull-ci-openshift-origin-master-e2e-gcp-ovn-upgrade/1633489612569579520

Reply:
Leaving the test as a flake may prove useful in that we can see how common it is on various nurps.